SegmentedControlTab.js

```diff
@@ -133,15 +146,19 @@ SegmentedControlTab.propTypes = {
     borderRadius: PropTypes.number,
     textNumberOfLines: PropTypes.number,
     allowFontScaling: PropTypes.bool,
+    accessible: PropTypes.bool,
+    accessibilityLabel: PropTypes.array,
 };
 
 SegmentedControlTab.defaultProps = {
     values: ['One', 'Two', 'Three'],
+    accessible: true,
+    accessibilityLabel: ['', '', ''],
```

Comment:
This should be removed as it's optional and this will lead to a `bug` setting accessibility to empty ` " "` when we don't use `accessibilityLabel` prop.

because as per the logic in 

```javascript
const getAccessibilityLabelByIndex = (accessibilityLabels, index) => {
  return accessibilityLabels && accessibilityLabels.length > 0 && accessibilityLabels[index] ?accessibilityLabels[index] : undefined
}
```
the length will always be 3 and `accessibilityLabels[index]` is empty `" "`.

Reply:
Whats wrong with having defaultProps for optional fields? If so, there are 19 other default props already in the code - hope they are optional too?

and there won't be any bug for when the consumer doesn't use `accessibilityLabel` prop. Check the below code snippet which covers the logic.

http://jsbin.com/sowukuwiyu/edit?js,console

will update the props `accessibilityLabels` and defaultProps as `accessibilityLabels: []`  to avoid returning the length as 3.

Reply:
perfect👍